### PR TITLE
Integration/1.0.3

### PR DESCRIPTION
**Pull request by sobyte**

### CODE REVIEW

File: Components/Hydrator/ResourceHydrator/CustomerHydrator/PrivateCustomerHydrator.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ public function hydrateOrFetch(
         $shippingAddress = $data['shippingaddress'];
         $billingAddress  = $data['billingaddress'];
 
-        $phoneNumber = \preg_replace(self::PHONE_NUMBER_REGEX, '', $billingAddress['phone']);
+        $phoneNumber = \preg_replace(self::PHONE_NUMBER_REGEX, '', $billingAddress['phone'] ?? '');
```
Comment (sixer1182) [inline Components/Hydrator/ResourceHydrator/CustomerHydrator/PrivateCustomerHydrator.php:30]: This seems to be missing in the changelog.

File: plugin.xml [hunk 2]
```diff
@@ -157,6 +157,30 @@
         </ul>
         ]]>
     </description>
+
```
Comment (sixer1182) [inline plugin.xml:160]: The following Item seems to be missing here: "Fix used translations in backend"

**Review by sixer1182 — COMMENTED** [review]
LGTM
just two questions for understanding.
See inline comments.

File: Components/Hydrator/ResourceHydrator/CustomerHydrator/PrivateCustomerHydrator.php (hunk 1 above)
Comment (macidev) [reply Components/Hydrator/ResourceHydrator/CustomerHydrator/PrivateCustomerHydrator.php:30]: This change was done during the compatibility for PHP 8 and implements a NULL check

**Review by sixer1182 — APPROVED** [review]

**Pull request merged**